packages/edit-site/src/components/page-templates/dataviews-templates.js

```diff
@@ -221,6 +239,21 @@ export default function DataviewsTemplates() {
 			} );
 		}
 
+		// Handle filters.
+		if ( view.filters.length > 0 ) {
+			view.filters.forEach( ( filter ) => {
+				if (
+					filter.field === 'author' &&
+					filter.operator === OPERATOR_IN &&
+					filter.value !== ''
```

Comment:
Do we need to check explicitly for empty string?

Reply:
I updated the condition to check if the filter has value instead of checking for an empty string.